components/organisms/o-login.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <div class="o-login">
+    <SfModal :visible="!!activeElem" @close="switchElem(null)">
+      <transition name="fade" mode="out-in">
+        <div v-if="activeElem === 'login'" key="log-in">
```

Comment:
co sądzisz o tym żeby wyciągnąć to do osobnego komponentu? :thinking: Podobnie można zrobić z formularzem rejestracji. czyli mielibyśmy coś takiego
```
<m-form-login  v-if="activeElem === 'login'" key="log-in" />
<m-form-register  v-if="activeElem === 'register'"  key="sign-up" class="form" />
```
Chyba nawet nie trzebaby przekazywać propsów, bo praktycznie komponenty mają całą potrzebną logikę. Jakby ktoś chciał zrobić np Page zamiast Modala to wystarczy że zaimportuje komponenty z gotowymi formularzami. No i ostatnia korzyść to mniejsze komponenty :) 
wtedy taki organism można nazwać `o-form-auth`

Reply:
Chciałem uniknąć tworzenia zbyt wielu komponentów ale masz rację. Komponenty logowania i rejestracji często mogą być wykorzystywane oddzielnie więc dobrze jest zrobić je od razu jako osobne komponenty. Zmieniłem to